app/services/cards.ts

```diff
@@ -0,0 +1,259 @@
+import axios from 'axios';
+import config from '../../config';
+import { externalApiError } from '../errors';
+
+export const BASE_URL = config.hearthstone.baseURL as string;
```

Comment:
I think that if in the Iconf interface you put these fields as a string you don't need to do this

Reply:
Why export?

Reply:
deleted export and " as string"